aptos/token_bridge/sources/structs/transfer.move

```diff
@@ -13,7 +13,7 @@ module token_bridge::transfer {
         deserialize_vector
     };
     use wormhole::cursor::{Self};
-
+    use wormhole::external_address::{ExternalAddress, from_vector, get_bytes};
```

Comment:
I think it's a little easier to read code if the import statement only grabs types unqualified, but not functions, so
```suggestion
    use wormhole::external_address::{Self, ExternalAddress};
```

because then looking at the code later, `external_address:get_bytes(...)` is clearer than just `get_bytes(...)`

Reply:
done